paper-toast.html

```diff
@@ -170,9 +170,16 @@
         },
 
         /**
-         * Show the toast. Same as `open()` from `IronOverlayBehavior`.
+         * Show the toast. Without arguments, this is the same as `open()` from `IronOverlayBehavior`.
          */
-        show: function() {
+        show: function(properties) {
+          properties = properties || {};
+          if (typeof properties == 'string') {
+            this.text = properties;
+          } else {
+            var t = this;
+            Object.keys(properties).forEach(function(key){ t[key] = properties[key]; });
```

Comment:
I tend to prefer the more conservative approach, only handle `text` and `duration`.
@cdata WDYT?